src/dialogs/more-info/controls/more-info-media_player.ts

```diff
@@ -277,17 +277,6 @@ class MoreInfoMediaPlayer extends LitElement {
         : undefined;
     }
 
-    if (state === "idle") {
-      return supportsFeature(stateObj, SUPPORTS_PLAY)
-        ? [
-            {
-              icon: "hass:play",
-              action: "media_play",
```

Comment:
So instead of just a play button when idle, we now show the power, next, previous, and play button? I don't think that is what we want?

Reply:
It is now consistent across all three usages: Entity row, more-info and media card. 

And yes, if the player supports "next" and "previous" they are shown together with the "play" button, since those can also act as play buttons. E.g. on the Spotify integration, if the player is idle and you press "next", then the next song will start playing. So it made sense for me to group them. Does it also for you or would you propose to not show them in idle/on?

Reply:
Idle is not paused right?

Reply:
No, those are separate. Basically "idle" is the default for players as a service (such as Spotify) that cannot be turned off.

Reply:
Then I'm not sure what next would do. Next of what? If I'm not currently playing anything?

Reply:
If you are not playing anything, there is still a selection in Spotify from the last time you played something, so "next" is whatever Spotify has queued up as next title (e.g. next song in the playlist).
![image](https://user-images.githubusercontent.com/114137/97619763-e6e56100-1a20-11eb-93ff-b22bb73bd81b.png)

Reply:
Your screenshot is state `paused` I think, can you check your state? Idle doesn't show media information.

Reply:
Just as a note: Aligned with Bram that idle will not show previous/next buttons.